src/client/ui/styles.less

```diff
@@ -750,6 +750,7 @@
             text-overflow: ellipsis;
             font-family: monospace !important;
             font-size: 12px;
+            color: black;
```

Comment:
Why we add `color:black` in the `.selector-value` class but not in the `.selectors-list` parent?

Reply:
updated